### DISCUSSION TITLE
add device from Ikea

### DISCUSSION THREAD

**Discussion by daanbaanvinger** [Koenkk/zigbee2mqtt#21907] [Devices]
I'm quite new to all this and I haven't managed to add an unsuported Ikea lamp yet. I did find the 'external definition' under 'Dev console' and I was wondering if this could be added?

**Comment by daanbaanvinger** [comment]
const {deviceEndpoints, light} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['TRADFRI bulb E14 CWS globe 806lm'],
    model: 'TRADFRI bulb E14 CWS globe 806lm',
    vendor: 'IKEA of Sweden',
    description: 'Automatically generated definition',
    extend: [deviceEndpoints({"endpoints":{"1":1,"242":242}}), light({"colorTemp":{"range":[250,454]},"color":{"modes":["xy","hs"],"enhancedHue":true}})],
    meta: {"multiEndpoint":true},
};

module.exports = definition;

**Reply by Koenkk** [reply]
What is the model number of this device? I expect something like `LED1738G7`

**Reply by daanbaanvinger** [reply]
![image](https://github.com/Koenkk/zigbee2mqtt/assets/59261437/0be50d36-8560-4dbb-87c2-3625d1db5511)

**Reply by Koenkk** [reply]
I mean the one written on the device itself or the product box.

**Reply by daanbaanvinger** [reply]
Oh sorry I didn't realize that, the model number is: LED2111G6

**Reply by Koenkk** [reply]
It was already added in https://github.com/Koenkk/zigbee-herdsman-converters/commit/09d4e69033e8e9daa2734ee554b36556a2f4a830, will be supported in the 1 April release
